client/js/components/month/month.js

```diff
@@ -1,12 +1,34 @@
 /** @jsx React.DOM */
 var React = require('react');
-var MonthCalendarContainer = require('./month-calendarcontainer.js');
+var MonthEventsContainer = require('./month-eventcontainer.js');
```

Comment:
Not sure why we are switching the components name between MonthEventContainer and MonthsEventContainer as we required it in?